build-data-capturing-gradle-samples/capture-maxparallelforks/gradle-maxparallelforks.gradle

```diff
@@ -0,0 +1,10 @@
+def buildScan = project.extensions.findByName('buildScan')
+if (!buildScan) {
+    return
+}
+
+gradle.afterProject {
```

Comment:
Why `gradle.afterProject`? I don't think any of the other build config samples do that.

Reply:
I ported it from the [original implementation in the CCUD plugin](https://github.com/gradle/common-custom-user-data-gradle-plugin/blob/main/src/main/java/com/gradle/CustomBuildScanEnhancements.java#L465-L472), but yeah I don't think it is strictly necessary to do it that way

Reply:
Yep makes sense. On it 👍